inst/rstudio/templates/project/skeleton.dcf

```diff
@@ -1,4 +1,11 @@
-Title: Book Project using bookdown
-Binding: bookdown_skeleton
-OpenFiles: README.md, _bookdown.yml, _output.yml, index.Rmd
+Title: Book project using bookdown
+OpenFiles: README.md, index.Rmd, _bookdown.yml, _output.yml
```

Comment:
I think I meant to open `index.Rmd` last so it will be the first file in the editor. The order of files were changed here, and I guess users will first see `_output.yml`?

Reply:
They may have changed something but when I try in the IDE stable release (1.4.1717), the first file in the list is the one that end up activated, so README.md

The order is then the order of the file in the source pane.

Reply:
Okay. That sounds good to me.